blueprint/api.apib

```diff
@@ -2565,6 +2712,47 @@ This endpoint allows you to check whether a username is valid (by running a vali
 + data(array[Stripe Plan Resource], required)
 + include JSON API Version
 
+## Stripe Subscription Attributes (object)
++ `cancelled-at`: null (string) - If the subscription has been canceled, the date of that cancellation.
++ created: `2016-07-08T03:03:51.967Z` (string)
```

Comment:
I would add the description we have for `created` elsewhere, specifying this is the date on Stripe's system.